dev/cf_deployment/bump/set_release_urls.sh

```diff
@@ -5,12 +5,12 @@ set -o errexit -o nounset
 cf_deployment_yaml="deploy/helm/scf/assets/cf-deployment.yml"
 dest="deploy/helm/scf/assets/operations/set_release_urls.yaml"
 
-printf "# This ops file sets the Quarks images for the releases.\n\n" > "${dest}"
+printf "# This ops file sets the Quarks images for the releases.\\n\\n" > "${dest}"
```

Comment:
FWIW, I feel like using single quotes for the formatting string is clearer. But double quotes and escaping the backslash is fine too.